src/main/starlark/builtins_bzl/common/builtin_exec_platforms.bzl

```diff
@@ -130,6 +130,28 @@ bazel_fragments["AndroidConfiguration.Options"] = fragment(
         "//command_line_option:internal_persistent_busybox_tools",
         "//command_line_option:internal_persistent_multiplex_busybox_tools",
         "//command_line_option:incompatible_disable_native_android_rules",
+        "//command_line_option:android_databinding_use_androidx",
```

Comment:
Hi @fmeum . We're trying to get this merged for Bazel 8.

Why were these flags added? Does this have something to do with the `experimental_android_databinding_v2`below?

Reply:
It's possible that they have been deleted in the meantime and I just didn't notice during the rebase. I didn't mean to add new flags, I just wanted to propagate all existing experimental flags.